src/otk_osbuild/command.py

```diff
@@ -191,3 +183,15 @@ def file_from_path():
             }
         )
     )
+
+
+def root():
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers()
+
+    (subparsers.add_parser('depsolve_dnf4')).set_defaults(func=depsolve_dnf4)
+    (subparsers.add_parser('file_from_text')).set_defaults(func=file_from_text)
+    (subparsers.add_parser('file_from_path')).set_defaults(func=file_from_path)
```

Comment:
If we're changing these to be underscores then the mapping also needs to be updated in `src/otk/external.py` at the bottom.

Reply:
If the naming convention has changed? I keept the original function names. Shall I convert them to `_depsolve_dnf4` etc?